cob_mecanum_controller/CMakeLists.txt

```diff
@@ -0,0 +1,58 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(cob_mecanum_controller)
+
+add_compile_options(-std=c++14)
```

Comment:
PR looks ok in general...but I'm not sure whether c++14 is allowed by the buildfarm for kinetic...
could you investigate and clarify with a link?

do you run raw-mini under kinetic or melodic?

Reply:
we decided to upgrade raw-mini to melodic --> only melodic is needed where c++14 is officially supported

Reply:
will I need a "dummy package" for the pipelines to succeed (or strip out the c++14 features)?

Reply:
I don't know the restrictions of the buildfarm
maybe you could try the prerelease test or the respective feature of industrial_ci...

Reply:
even if raw-mini is melodic only, we would need a kinetic release of this package in order to satisfy dependencies of cob_robot's kinetic_dev branch...

I don't want to create a melodic_dev branch for cob_robots just because of this...